utils/s2n_safety.h

```diff
@@ -133,6 +133,26 @@
  */
 #define ENSURE_PTR_NONNULL( x )                     ENSURE_PTR((x) != NULL, S2N_ERR_NULL)
 
+/**
+ * Ensures the `condition` is `true`, otherwise the function will `BAIL` with a `S2N_ERR_SAFETY` error
```

Comment:
Oh shoot... I just realized the docs are incorrect :)

Should be `S2N_ERR_PRECONDITION_VIOLATION`, not `S2N_ERR_SAFETY`

Reply:
I missed it too! Just fixed tho.